modules/api/src/main/java/org/apache/ignite/app/Ignition.java

```diff
@@ -55,4 +55,11 @@
      * @return Started Ignite node.
      */
     public Ignite start(@NotNull String name, @NotNull Path workDir);
+
+    /**
+     * Stops node with given node. It's possible to stop both already started node or node that is currently starting.
```

Comment:
Here and below: `Stops a node with the given {@code name}.`

Reply:
Fixed.